internal/shim-sev/src/frame_allocator.rs

```diff
@@ -23,15 +24,12 @@ use x86_64::{align_down, align_up, PhysAddr, VirtAddr};
 #[allow(clippy::integer_arithmetic)]
 pub struct Page2MiB([u8; bytes![2; MiB]]);
 
-lazy_static! {
-    /// The global ShimFrameAllocator RwLock
-    pub static ref FRAME_ALLOCATOR: RwLock<FrameAllocator> = {
-        RwLock::<FrameAllocator>::const_new(
-            spinning::RawRwLock::const_new(),
-            unsafe { FrameAllocator::new() },
-        )
-    };
-}
+/// The global ShimFrameAllocator RwLock
+pub static FRAME_ALLOCATOR: Lazy<RwLock<FrameAllocator>> = Lazy::new(|| {
+    RwLock::<FrameAllocator>::const_new(spinning::RawRwLock::const_new(), unsafe {
```

Comment:
Could you use the same formatting breaks as the old code? It was easier to read.

Reply:
Well, `rustfmt` does all the stuff, which it didn't inside the `lazy_static!` macro call.
I can try to persuade `rustfmt` with local vars to format it more readable.